python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -34,6 +34,7 @@
 
 import lsst.log
 import lsst.ap.pipe as apPipe
+import lsst.ap.pipe.make_ppdb as makePpdb
```

Comment:
Do you really need to rename it? Whey not `from lsst.ap.pipe.make_ppdb import makePpdb` or `from lsst.ap.pipe import make_ppdb`?